wls-gui-wahllokalsystem/tests/composables/user/userService.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { useUserTestDataFactory } from "@tests/utils/common/UserTestDataFactory.ts";
+import { beforeEach, describe, expect, it, vi } from "vitest";
+
+import { useUserService } from "@/composables/user/userService.ts";
+import { User } from "@/types/User.ts";
+
+const mockDefinitions = vi.hoisted(() => ({
+  user: vi.fn(),
+  toModel: vi.fn(),
+}));
+
+vi.mock("@/api/wls-clients/generated-auth-api", () => ({
+  UserControllerApi: vi.fn().mockImplementation(() => ({
+    user: mockDefinitions.user,
+  })),
+  Configuration: vi.fn(),
+}));
+vi.mock("@composables/user/userMapper.ts", () => ({
```

Comment:
sonst greift der Mock nicht

```suggestion
vi.mock("@/composables/user/userMapper.ts", () => ({
```

Reply:
done [1bee407](https://github.com/it-at-m/Wahllokalsystem/pull/1297/commits/1bee4078ecf1e6d0a82142268c8ad3618f7cbd3d)